oam/install_scripts/mcs-stop-controllernode.sh.in

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+kill -15 $(pidof workernode)
+kill -15 $(pidof controllernode)
```

Comment:
We have to explicitly call SIGTERM on workernode, otherwise controllernode will get stopped first before workernode gets signaled to shutdown due the dependency chain running in reverse on stop/restart